sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/registry.py

```diff
@@ -230,5 +231,6 @@ def _to_rest_object(self) -> RestRegistry:
                 managed_resource_group=self.managed_resource_group,
                 ml_flow_registry_uri=self.mlflow_registry_uri,
                 region_details=replication_locations,
+                managed_resource_group_tags=self.tags,
```

Comment:
So `tags` are duplicated as managed resource group tags too?  Can you write a comment for the justification?

Reply:
Added a justification in the file comments. TL;DR it's really important for janitor policies, and might become forced behavior in the future.